lib/rules/_base.js

```diff
@@ -521,16 +522,32 @@ module.exports = class {
       defaults.filePath = this.filePath;
     }
 
-    // perform the node prop expansion only if those properties don't exist in result
+    if (!result.node) {
+      if (!loggedRules.has(this.ruleName)) {
+        let message = `Calling the log method without passing the node is deprecated. Please ensure you pass node in the log method's result.`;
+
+        if (process.env.JEST_WORKER_ID !== undefined) {
```

Comment:
This was to track for local dev, but as we've discussed this isn't likely a good solution, since some plugins can _not_ use jest. Changing this to a custom env var, which we'll opt into in our own test harness.